### DISCUSSION TITLE
Is there any way to mass rename assets with the same name as their filenames?

### DISCUSSION THREAD

**Discussion by AronRock** [Gorgious56/asset_browser_utilities#90] [Q&A]
That's all. 

I also wanna thank you for making this great addon, you made a life saving tool!

**Comment by Gorgious56** [comment]
Hello !

This should be available in the latest release I just published. 

https://github.com/Gorgious56/asset_browser_utilities/releases/tag/0.2.4

Got To Asset > Custom operation and on the bottom should Rename, as Blend File

![image](https://user-images.githubusercontent.com/25156105/184475612-6045f79c-8076-4ffb-8406-e9c4855fbc9b.png)

![image](https://user-images.githubusercontent.com/25156105/184475628-c2e3fc90-4358-4b0f-bfb6-cb2cba7af3c5.png)

Note that two assets that are located in the same file that belong to the same container (objects, collections, materials, etc.) can't share the same name. Otherwise .001, .002 etc. suffixes will be added

**Reply by AronRock** [reply]
Thank you so much! You're a hero.

**Comment by MadeOfClicks** [comment]
Great work,I'll soon be ready to test out the new features, thanks for continuing to work on the addon and sharing it!Best regards Am 13.08.2022 10:24 schrieb Gorgious56 ***@***.***>:
Hello !
This should be available in the latest release I just published.
https://github.com/Gorgious56/asset_browser_utilities/releases/tag/0.2.4
Got To Asset > Custom operation and on the bottom should Rename, as Blend File


Note that two assets that are located in the same file that belong to the same container (objects, collections, materials, etc.) can't share the same name. Otherwise .001, .002 etc. suffixes will be added
